util/cosmos/misc.js

```diff
@@ -11,14 +11,17 @@ export function assertOk (res) {
     throw new Error(res.error)
   }
 
+  const txRes = res.tx_response
   // Sometimes we get back failed transactions, which shows only by them having a `code` property
-  if (res.code) {
-    const message = JSON.parse(res.raw_log).message
+  if (txRes.code) {
+    const message = txRes.raw_log.message
+      ? JSON.parse(txRes.raw_log).message
+      : txRes.raw_log
```

Comment:
if you can access `txRes.raw_log.message` you don't have to do `JSON.parse`..
if its plaintext, we can't access `.message` until we `JSON.parse`
just use `txRes.raw_log` i guess